Makefile

```diff
@@ -53,12 +53,16 @@ lint:
 
 .PHONY: build
 build: deps
-	$(GO) install -x -ldflags "$(GOBUILD_LDFLAGS)" $(ALL_PACKAGES)
+	GOOS=linux $(GO) build -o gfmrun_linux_$(VERSION_VALUE)/gfmrun  -x -ldflags "$(GOBUILD_LDFLAGS)" ./cmd/gfmrun/main.go
+	tar -czf gfmrun_linux_$(VERSION_VALUE).tar.gz gfmrun_linux_$(VERSION_VALUE)
+	GOOS=darwin $(GO) build -o gfmrun_darwin_$(VERSION_VALUE)/gfmrun  -x -ldflags "$(GOBUILD_LDFLAGS)" ./cmd/gfmrun/main.go
+	tar -czf gfmrun_darwin_$(VERSION_VALUE).tar.gz gfmrun_darwin_$(VERSION_VALUE)
+	GOOS=windows $(GO) build -o gfmrun_windows_$(VERSION_VALUE)/gfmrun.exe  -x -ldflags "$(GOBUILD_LDFLAGS)" ./cmd/gfmrun/main.go
+	tar -czf gfmrun_windows_$(VERSION_VALUE).tar.gz gfmrun_windows_$(VERSION_VALUE)
```

Comment:
tar.gz is not very useful on windows.
Also, I wonder if we should just upload  executables rather than tar.gzping them?

It's a single file, so tar is pointless, and gzip is not very useful for binary files as well.

Reply:
Good Idea. Let me update it.